include/gunrock/applications/async/util/time.cuh

```diff
@@ -0,0 +1,29 @@
+
+/******************************************************************************
+ * Timing
```

Comment:
What's different between `time.cuh` and https://github.com/bkj/essentials/blob/master/include/gunrock/util/timer.hxx?

Reply:
nothing, can swap